### DISCUSSION TITLE
Make it possible to backup one's configuration on one's hard drive

### DISCUSSION THREAD

**Discussion by liar666** [dailydotdev/daily#994] [Feature Requests]
Hi,

It would be great if one could backup one's configuration on one's hard drive.

It could then be used:
- to reimport it later in the same account, without having to reconfigure the whole thing
- to transfer to another account, for instance for a friend, or if you want to transfer from a Google-created account to an Email-created account [which is impossible at the moment: https://github.com/dailydotdev/daily/issues/253#issuecomment-1778883726]
- to configure an external RSS reader with the same feeds.

**Comment by idoshamun** [comment]
Hi @liar666,

The configuration is synced across devices as long as you use the same account.
We're considering what will be the best way to let users invite other users with the same configuration.
As for supporting external RSS readers, it's currently out of our scope as we also scrape non-RSS blogs.
